local-cli/cli.js

```diff
@@ -14,7 +15,8 @@ function printUsage() {
     '',
     'Commands:',
     '  start: starts the webserver',
-    '  install: installs npm react components'
+    '  install: installs npm react components',
+    '  build: builds the javascript bundle for offline use'
```

Comment:
Let's keep it `bundle`, we are currently thinking about new `build` command that would compile ObjC code so you don't even have to run Xcode